scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -29,7 +29,7 @@
 from .process import set_column_map
 
 # These directories are excluded from testing
-_EXCLUDE_DIRS = ['common', 's2201', 's0902', '__pycache__', '~']
+_EXCLUDE_DIRS = ['common', '__pycache__', '~']
```

Comment:
`s2201` should be a part of this list since it has its own test.

Reply:
Thanks for the review! I have addressed these comments.